# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../oaf10/bison_syntax.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ru oaf-0.6.10_/oafd/ac-query-parse.y oaf-0.6.10/oafd/ac-query-parse.y
--- oaf-0.6.10_/oafd/ac-query-parse.y	2001-10-24 23:15:13.000000000 +0200
+++ oaf-0.6.10/oafd/ac-query-parse.y	2002-11-16 14:35:53.000000000 +0100
@@ -155,7 +155,7 @@
 stringlist: P_CONST_STRING { $$ = g_slist_prepend (NULL, $1); }
 	| stringlist COMMA P_CONST_STRING { $$ = g_slist_append ($1, $3); };
 
-expr_variable: P_DOLLAR P_CONST_ID { $$ = qexp_variable_new ($2); }
+expr_variable: P_DOLLAR P_CONST_ID { $$ = qexp_variable_new ($2); };
 
 expr_function: P_CONST_ID LPAREN exprlist RPAREN { $$ = qexp_function_new ($1, $3); }
 	| P_CONST_ID LPAREN RPAREN { $$ = qexp_function_new ($1, NULL); }
Only in oaf-0.6.10/oafd: ac-query-parse.y~
